### PR TITLE
make parcel build the source code

### PR DESCRIPTION
**Pull request by ysz**
This what makes parcel build the source code without modifying the modules files.

It builds both web and extension (but with stubs for background pages etc. at the moment)

```
$ yarn parcel
yarn run v1.22.17
$ parcel build
✨ Built in 8.68s

dist/web/index.html                        969 B    1.84s
dist/web/index.7ca3e8c9.js           ⚠️  3.89 MB    7.20s
dist/web/web-vitals.baf1c487.js          4.74 KB    281ms
dist/web/index.edd668fd.css               2.5 KB    588ms
dist/web/index.6b05ee56.js           ⚠️  3.79 MB    7.22s
dist/web/web-vitals.0ae59b28.js          4.72 KB    281ms
dist/ext/manifest.json                   1.49 KB    627ms
dist/ext/static/img/oasis.png            3.51 KB    1.80s
dist/ext/static/popup.html                 369 B    465ms
dist/ext/static/popup.bf1a20d9.js          226 B    396ms
dist/ext/static/popup.6b78fafd.js          226 B    395ms
dist/ext/background.js                     198 B     86ms
✨  Done in 9.37s.
```

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,36 @@
 {
-  "short_name": "Oasis Wallet",
```
Comment (lukaw3d) [inline public/manifest.json:2]: Maybe the old manifest.json can be kept and renamed to web-app-manifest.json to reduce confusion

File: src/app/components/DateFormatter/index.tsx [hunk 2]
```diff
@@ -9,14 +9,20 @@ interface Props {
...
+
 const dateFormat = new Intl.DateTimeFormat(undefined, {
   year: 'numeric',
   month: 'numeric',
   day: 'numeric',
   hour: 'numeric',
   minute: 'numeric',
   second: 'numeric',
-  timeZone: process?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,
+  timeZone: getTimeZone(),
```
Comment (lukaw3d) [inline src/app/components/DateFormatter/index.tsx:25]: This is really strange. I can build:
- your version
- `timeZone: process?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,` iff I keep getTimeZone defined above
- `timeZone: globalThis?.env?.NODE_ENV === 'test' ? 'UTC' : undefined,`

Comment (lukaw3d) [reply src/app/components/DateFormatter/index.tsx:25]: I think we can just use `timeZone: process.env.NODE_ENV === 'test' ? 'UTC' : undefined`

Comment (ysz) [reply src/app/components/DateFormatter/index.tsx:25]: Thank you for taking a look @lukaw3d !

It has to do with 

https://github.com/parcel-bundler/parcel/issues/7101

https://github.com/parcel-bundler/parcel/issues/4458

**Comment by tjanez** [comment]
Is there anything to extract from this PR now that #746 is being merged?

**Pull request closed without merging**